app/routes/index.tsx

```diff
@@ -25,8 +30,72 @@ export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
   }
 }
 
+enum LoadMoreType {
+  disabled = 'disabled',
+  infini = 'infini',
+  button = 'button',
+  buttonInfini = 'buttonInfini',
+}
+
 export const shouldRevalidate: ShouldRevalidateFunction = () => false
 
+const Bottom = ({
+  isSingleQuestion,
+  fetchMore,
+}: {
+  isSingleQuestion: boolean
+  fetchMore: () => Promise<any>
+}) => {
+  const [remixSearchParams] = useSearchParams()
+  const urlLoadType = useCallback(() => {
+    const more = remixSearchParams.get('more')
+    if (more) return LoadMoreType[remixSearchParams.get('more') as keyof typeof LoadMoreType]
```

Comment:
the `more` url param can be used to manually specify what way of loading more questions should be used